tests/NLog.UnitTests/Layouts/JsonLayoutTests.cs

```diff
@@ -70,9 +70,9 @@ public void JsonLayoutRenderingAndEncodingSpecialCharacters()
             {
                 Attributes =
                     {
-                        new JsonAttribute("date", "${longdate}"),
-                        new JsonAttribute("level", "${level}"),
-                        new JsonAttribute("message", "${message}"),
+                        new JsonAttribute("date", "${longdate}", true),
+                        new JsonAttribute("level", "${level}", true),
+                        new JsonAttribute("message", "${message}", true),
```

Comment:
Please don't change existing unit tests, without good reason. (this is also a hint that it isn't backwards compatible)